minio/helpers.py

```diff
@@ -375,6 +375,19 @@ def is_non_empty_string(input_string):
 
     return True
 
+def is_valid_bucket_policy_name(policy):
+    """
+    Validates bucket policy
+    :param policy: S3 style Bucket policy.
+    :return: True if input is a valid policy structure.
```

Comment:
`:return: True if input is a valid policy structure.`
=>
`:return: True if policy parameter is of a valid type, 'string'.`